### DISCUSSION TITLE
Status squeue fails to execute, no error message attached

### DISCUSSION THREAD

**Discussion by Sanderson1887** [deepmodeling/dpgen#1628] [Q&A]
**Description**
I'm trying to replicate the dpgen tutorial on an HPC cluster to familiarize myself with the program. I followed the [Hands-On DPGen tutorial](https://tutorials.deepmodeling.com/en/latest/Tutorials/DP-GEN/learnDoc/DP-GEN_handson.html). The only major changes I made were to the machine.json file, where I updated the train, model_deviation, and first principles calculations to all be run on a SLURM HPC cluster. 
**Issue description**
Shortly after submitting the job, the job terminates after no more than 20 seconds. The job hash folder is created at the remote root correctly, and contains the [hash].sub, [hash].sub.run, [hash].json, along with the '000' folder and the 'data.init' folder. The submission folder does not contain anything else however. Attached here are my machine.json file and the error output.
[machine.json](https://github.com/user-attachments/files/16808672/machine.json)
[error.txt](https://github.com/user-attachments/files/16808687/error.txt)
Further, attempting to run 'dpdisp submission --download-terminated-log [hash]' said the error log was downloaded to a file, but did not list what file it was downloaded to (if any).
**Attempted Fixes**
The error occurring so quickly after submission and having trouble parsing the relevance of the errors given means I haven't had much luck troubleshooting this error. I see 4 similar errors in the discussion forum, but reading them has not been enlightening for my error.
**Versions**
Python 3.11.9
dpgen 0.12.1
DeePMD-kit 2.2.10

**Comment by njzjz** [comment]
The error message is wrong, which I have fixed in https://github.com/deepmodeling/dpdispatcher/pull/483. The actual executed command is `sbatch xx.run`. Do you get any error message or non-zero exit code when you manually execute the command?

**Reply by Sanderson1887** [reply]
Yes, DPkit executes correctly when I manually submit [hash].job. After updating dpdispatcher the [error.txt](https://github.com/user-attachments/files/16825805/error.txt) points to an error with 'cd $REMOTE_ROOT && sbatch [hash].sub'. 
Running the above error on an interactive compute node reveals that the error comes from sbatch not being available to call from a compute node. It looks like I'll have to rethink where I call dpgen.
